homeassistant/components/flume/sensor.py

```diff
@@ -137,26 +148,49 @@ def available(self):
 
     @property
     def unique_id(self):
-        """Device unique ID."""
-        return self._device_id
+        """Flume query and Device unique ID."""
+        return f"{self._flume_query_sensor}_{self._device_id}"
```

Comment:
Since we're changing the unique id also for the existing sensor this will be a breaking change. The users will need to clean up the stale entity from the entity registry via the GUI.

Reply:
Updated the type of change to breaking change.